packages/flutter_test/lib/src/matchers.dart

```diff
@@ -354,6 +354,57 @@ Matcher coversSameAreaAs(Path expectedPath, { required Rect areaToCompare, int s
 /// ```
 /// {@end-tool}
 ///
+/// Fonts can be rendered differently according with the platform or the Flutter
+/// version in which the test is run. While the difference may be minimal, it's
+/// still valuable to make the test fail. For example, a golden file generated
+/// on Windows is going to differ from the one produced by another operating
+/// system. Even on the same platform, if the generated golden is tested with a
+/// different Flutter version, the test may fail.
```

Comment:
This paragraph about caveats seems better placed at the end. Can you also add a heading for this section?

```suggestion
/// ## Including Fonts
///
/// Custom fonts may render differently across different platforms, or
/// between different versions of Flutter. For example, a golden file generated
/// on Windows with fonts will likely differ from the one produced by another
/// operating system. Even on the same platform, if the generated golden is
/// tested with a different Flutter version, the test may fail and require an updated
/// image.
```